src/pypi_changes/_cli.py

```diff
@@ -43,6 +43,8 @@ def _define_cli_arguments() -> ArgumentParser:
     cache_help = "seconds how long requests should be cached (pass 0 to bypass the cache, -1 to cache forever)"
     parser.add_argument("--cache-duration", "-d", default=3600, type=int, help=cache_help, metavar="SEC")
 
+    parser.add_argument("--json", action="store_true", default=False, help="Show output in JSON format")
```

Comment:
Would prefer to make the presentation a mutually exclusive flag combination, where the default is the current state so it's easily extensible.

Reply:
Makes sense. How do you want to call that flag? `--output`, `--format`? And how do you want to call the current format that uses rich for console?

Reply:
Done